### PR TITLE
Clean up contributing info

### PR DESCRIPTION
**Pull request by mDuo13**
- Incorporate translations from #2076
- Replace duplicate/outdated contents of `CONTRIBUTING.md` / `CONTRIBUTING.ja.md` files with a link to where the updated contributing docs will be.
- Update "Contribute Translations" so it doesn't suggest translating a removed page.
- Rename `what-is-xrpl.md` to match the page title
- Fix `parent:` info in frontmatter and remove blank line after frontmatter, for both "What is the XRP Ledger?" and "What is XRP?"

### CODE REVIEW

**Comment by mDuo13** [comment]
@develoQ I can't tag you as a reviewer, but please take a look.

File: content/resources/contribute-documentation/documentation-translations.ja.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        prefix: "/"
+    -   code: ja
+        display_name: 日本語
+        prefix: "/ja/"
+```
+
+同じ`dactyl-config.yml`ファイルには、XRP Ledger Dev Portalの各コンテンツページのエントリがあります。ページが翻訳されている場合、各翻訳ごとに個別の項目があり、その翻訳の“ターゲット"にリンクされています。ページがまだ翻訳されていない場合、すべてのターゲットで英語版が使用されます。
+
+慣習上、ページのHTMLファイル名（`html`フィールド）はすべての言語バージョンで同じであるべきです。翻訳されたMarkdownのソースファイルは英語版と同じファイル名を使用すべきです。ただし、拡張子は英語版の`.md`だけではなく、`.{言語コード}.md`を使用すべきです。例えば、日本語に翻訳されたファイルの拡張子は`.ja.md`です。
+- **`blurb`** - ページの短い要約です。このテキストは主にカテゴリーのランディングページで使用されます。
```
Comment (tequdev) [inline content/resources/contribute-documentation/documentation-translations.ja.md]:
```suggestion
同じ`dactyl-config.yml`ファイルには、XRP Ledger Dev Portalの各コンテンツページのエントリがあります。ページが翻訳されている場合、各翻訳ごとに個別の項目があり、その翻訳の“ターゲット"にリンクされています。ページがまだ翻訳されていない場合、すべてのターゲットで英語版が使用されます。(新しいページが英語のみ追加され、他の言語が提供されない場合、リンクチェッカーはそれをリンク切れとして報告します。)

ページを翻訳するということは、そのページのエントリを他の言語と分割するということです。ページのメタデータは`dactyl-config.yml`ファイルか、ページのMarkdownファイルの先頭にあるfrontmatterに設定します。

| フィールド | 備考 |
|----------|------|
| `html` | ページのHTMLファイル名。慣例により、これはすべての言語バージョンで同じであるべきです。 |
| `md` | ページのMarkdownソースファイル。翻訳されたMarkdownソースファイルは英語版と同じファイル名を使用してください。ただし、拡張子は英語版の`.md`ではなく、`.{言語コード}.md`を使用してください。例えば、日本語の翻訳ファイルの拡張子は `.ja.md` です。 |
| `blurb` | ページの簡単な要約。これは翻訳されるべきです。このテキストは、検索エンジン最適化のためのメタデータや、自動生成されるランディングページで使用されます。 |
```

**Review by tequdev — APPROVED** [review]
Since the English version has been updated, the Japanese version has also been made compatible.

**Review by amarantha-k — APPROVED** [review]
Reviewed updates other than the Japanese translation commit - looks good.

**Pull request merged**